iaso/models/base.py

```diff
@@ -149,6 +120,19 @@ class Account(models.Model):
     analytics_script = models.TextField(blank=True, null=True)
     custom_translations = models.JSONField(null=True, blank=True)
 
+    @property
+    def short_sanitized_name(self):
+        """
+        Short sanitized name mainly used in file path storage
+        """
+        text = self.name.lower()
+        text = re.sub(r"\s+", "_", text)
+        text = re.sub(r"[^a-z0-9_]", "", text)
+        text = re.sub(r"_+", "_", text)
+        text = text[:30]
+        text = text.strip("_")
+        return text if text else "invalid_name"
```

Comment:
```suggestion
    @property
    def short_sanitized_name(self):
        """
        Short sanitized name mainly used in file path storage
        """
        if not self.name or not self.name.strip():
            return "invalid_name"

        text = slugify(self.name, allow_unicode=False)
        text = text.replace('-', '_')[:30]

        return text if len(text) >= 2 else "invalid_name"
```

I would go for Django's slugify (`from django.utils.text import slugify`) because it's battle tested and it handles Unicode and special characters gracefully:

- it converts rather than strips: "café" → "cafe" (not "caf")
- another benefit: less custom regex logic to maintain